translate/storage/test_php.py

```diff
@@ -763,3 +755,8 @@ def test_simpledefinition_after_define(self):
         phpunit = phpfile.units[1]
         assert phpunit.name == "$lang['mediaselect']"
         assert phpunit.source == "Bestand selectie"
+
+    def test_quotes(self):
+        phpsource = b'<?php\n$txt[\'DISPLAYEDINFOS\'] = "<a href=\\"__PARAM2__\\">Modificar...</a>";\n'
```

Comment:
Is it not possible to use triple quoted string to avoid having `\n` in the middle of the string?